angular2/src/angular2TextMask.ts

```diff
@@ -33,6 +33,7 @@ export default class MaskedInputDirective {
   }
 
   writeValue(value: any) {
+    this.textMaskInputElement.update(value)
```

Comment:
Actually, never mind. It's fine. That's expected, even the React component has the same thing. The `update` methods will know that it is being called twice and will ignore the second call.